boards/arm/efr32_slwstk6061a/doc/efr32_slwstk6061a.rst

```diff
@@ -0,0 +1,188 @@
+.. _efr32_slwstk6061a:
+
+EFR32-SLWSTK6061A
+#################
+
+Overview
+********
+
+The EFR32 Flex Gecko Wireless Starter Kit SLWSTK6061A contains a Wireless
+System-On-Chip from the EFR32FG family built on an ARM® Cortex®-M4F processor
+with excellent low power capabilities.
+
+.. figure:: efr32-flex-gecko-starter-kit.png
+   :width: 375px
+   :align: center
+   :alt: EFR32-SLWSTK6061A
+
+   EFR32-SLWSTK6061A (image courtesy of Silicon Labs)
```

Comment:
I added this note below the board image to credit Silicon Labs, see https://www.silabs.com/community/general-interest/forum.topic.html/usage_of_board_pictu-ydFZ